src/databaseQueries/userRoleExtension.js

```diff
@@ -80,24 +80,26 @@ module.exports = class userRoleExtension {
 				}
 
 				// Execute query with optional sorting, limiting, and skipping
-				let userDocuments
+				let result
```

Comment:
@adithyadinesh0412 Please rename the variable to userRoleExtension , and test it well